### ISSUE TITLE
feat: Allow exporting to .docx and .html

### ISSUE THREAD

**Issue by dacoaster** [dacoaster/yattie#60]
# Feature Request

Extend document export with .docx and .html options.

**Comment by vijiboy** [comment]
especially "export to html" to play those local video files of screen recordings.

**Comment by dacoaster** [comment]
Good point @vijiboy.  This should be a simple enough change, so I will try to get it in the next release after the one going out this week.

**Comment by dacoaster** [comment]
I mocked this up and the results weren't very satisfying. The HTML documents required all of the assets to be kept alongside the file so it overall wasn't very portable.

I'm going to work on making this more portable/easy to share, but it's going to take a bit longer than expected. Hopefully on the order of a few weeks.

**Comment by vijiboy** [comment]
great to see your quick response !
> Good point @vijiboy. This should be a simple enough change, so I will try to get it in the next release after the one going out this week.  

seems not that simple enough. 
kudos to you for keeping the feature work, in-progress. 🤩 
I feel this "export to HTML for videos" will be a very useful feature. 👍

**Comment by dacoaster** [comment]
This will be addressed with #64 .  I'll keep it open until that is fixed!
